apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -77,14 +73,12 @@ class ConnectionInstructions extends React.Component {
       : this.renderWebSerialConnectButton();
 
     if (!shouldUseWebSerial()) {
-      return null;
+      return <p>{applabI18n.makerSetupWebSerialNotSupported()}</p>;
```

Comment:
Do you mind adding a screenshot of what this looks like?

Reply:
Added to the description!

Reply:
Nit: Maybe boldtype this sentence?